src/test/java/org/fundacionjala/pivotal/cucumber/steps/CommonSteps.java

```diff
@@ -0,0 +1,24 @@
+package org.fundacionjala.pivotal.cucumber.steps;
+
+import cucumber.api.java.en.Given;
+import org.fundacionjala.core.util.Environment;
+import org.fundacionjala.pivotal.pageobjects.dashboard.Dashboard;
+import org.fundacionjala.pivotal.pageobjects.login.SignInPage;
+
+public class CommonSteps {
+
+    private Dashboard dashboard;

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "dashboard" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)